charts/pwa/templates/prefetch-job.yaml

```diff
@@ -27,15 +27,7 @@ spec:
             - name: SECS
               value: {{ default 3600 .stop | quote }}
             args: [
-              "--timeout=15",
-              "--spider",
-              "--no-check-certificate",
-              "--retry-connrefused",
-              "--tries=5",
-              "--execute=robots=off",
-              "--recursive",
-              "--level=0",
-              "--no-verbose",
+              {{ default "\"--timeout=15\", \"--spider\", \"--no-check-certificate\", \"--retry-connrefused\", \"--tries=5\", \"--execute=robots=off\", \"--recursive\", \"--level=0\", \"--no-verbose\"" .args }},
```

Comment:
Providing a multiline string here would remove the indention of each line in the resulting file, so I decided to keep it in a single line. Maybe there could be a more beautiful solution

Reply:
Unfortunately I currently have no better idea. So I will merge it for testing as it is.